examples/plot_nii_to_graph.py

```diff
@@ -0,0 +1,201 @@
+"""
+.. _nii_to_graph:
+
+=========================================================
+Compute connecivity matrices and graph properties from nii files
+=========================================================
+The nii_to_graph pipeline performs graph analysis from functional MRI file
+in NIFTI format.
+
+The **input** data should be a preprocessed, and in the same space (e.g. MNI
+space) as the template used to define the nodes in the graph.
+"""
+
+# Authors: David Meunier <david_meunier_79@hotmail.fr>
+
+# License: BSD (3-clause)
```

Comment:
you can add a comment like:

```py
# sphinx_gallery_thumbnail_number = 2
```

to choose which of your figures to show as thumbnail in the sphinx gallery

Reply:
I did not know that, this is indeed much better!